UIforETW/UIforETWDlg.cpp

```diff
@@ -1101,7 +1909,8 @@ void CUIforETWDlg::OnLbnDblclkTracelist()
 {
 	int selIndex = btTraces_.GetCurSel();
 	// This check shouldn't be necessary, but who knows?
-	if (selIndex < 0 || selIndex >= (int)traces_.size())
+	//That comment IS WRONG! CListBox::GetCurSel MAY return -1! (LB_ERR, when no item is selected)

```

Comment:
It's possible, just don't click *an item* in the list.

https://www.youtube.com/watch?v=MiMaXdGbfOc